docs/source/npq/guidance.html.md

```diff
@@ -179,14 +179,14 @@ An example request body is listed below.
 #### Example request body:
 
 ```
-{
-  "data": {
-    "type": "npq-application-accept",
-    "attributes": {
-      "schedule_identifier": "npq-leadership-spring"
```

Comment:
`V3` includes `schedule_identifier` but `V1` and `V2` don't.
If we are focused on V3 we would need to just add the `funded_place` attribute.